paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundActivity.kt

```diff
@@ -93,9 +100,13 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity(), ExternalPay
     @OptIn(ExperimentalCustomerSessionApi::class)
     @Suppress("LongMethod")
     override fun onCreate(savedInstanceState: Bundle?) {
+        enableEdgeToEdge()
         super.onCreate(savedInstanceState)
 
         setContent {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+                window.isNavigationBarContrastEnforced = false
```

Comment:
`window.isNavigationBarContrastEnforced` ensures that the navigation bar has enough contrast when a fully transparent background is requested. 

https://developer.android.com/codelabs/edge-to-edge#2